docs/docs/reference/crd-reference/app.md

```diff
@@ -4,18 +4,14 @@ comments: true
 
 # KeptnApp
 
-A `KeptnApp` resource lists all the [workloads](https://kubernetes.io/docs/concepts/workloads/)
+A `KeptnApp` resource lists all the
+[workloads](https://kubernetes.io/docs/concepts/workloads/)
 that constitute a logical application.
-It contains information about:
-
-- All [workloads](https://kubernetes.io/docs/concepts/workloads/) and checks
-  that are associated with a Keptn application
-- A list of tasks and evaluations to be executed
-  pre- and post-deployment.
-- Tasks referenced by `KeptnApp` are defined in a
-  [KeptnTaskDefinition](taskdefinition.md) resource.
-  `KeptnApp` identifies the task by the value of the `metadata.name` field
-  and does not need to understand what runner is used to define the task.
+Keptn auto-generates it to work with the corresponding
```

Comment:
KeptnAppContext does not need to be in the system at all, it's more like an additional feature, I would leave  this sentence out to speak only about KeptnApp in this file

Reply:
reworded and done